CHANGELOG/CHANGELOG-1.15.md

```diff
@@ -0,0 +1,81 @@
+# Changelog 1.15
+
+## 1.15.0 - 2024-01-26
+
+### Added
+
+- Add a new feature `NodeNetworkPolicy` to allow user to apply `ClusterNetworkPolicy` to Kubernetes Nodes. ([#5658](https://github.com/antrea-io/antrea/pull/5658) [#5716](https://github.com/antrea-io/antrea/pull/5716), [@hongliangl] [@Atish-iaf])
+    * Refer to [this document](https://github.com/antrea-io/antrea/blob/release-1.15/docs/antrea-node-network-policy.md) for more information about this feature.
+- Add L7 network flow export support in Antrea to enable users to export L7 protocol information. ([#5218](https://github.com/antrea-io/antrea/pull/5218), [@tushartathgur])
+    * Refer to [this document](https://github.com/antrea-io/antrea/blob/release-1.15/docs/network-flow-visibility.md) for more information about this feature.
+- Add migration tool to support migrating from other CNIs to Antrea. ([#5677](https://github.com/antrea-io/antrea/pull/5677), [@hjiajing])
+    * Refer to [this document](https://github.com/antrea-io/antrea/blob/release-1.15/docs/migrate-to-antrea.md) for more information about this tool.
+- Support Egress using IPs from a different subnet from the default Node subnet. ([#5799](https://github.com/antrea-io/antrea/pull/5799), [@tnqn])
```

Comment:
s/from a different subnet from the default Node subnet/from a subnet that is different from the default Node subnet

Reply:
Updated.